dot/network/service.go

```diff
@@ -95,8 +95,8 @@ func NewService(cfg *Config, msgSend chan<- Message, msgRec <-chan Message) (*Se
 		syncer:       newSyncer(host, cfg.BlockState, cfg.SyncChan),
 		blockState:   cfg.BlockState,
 		networkState: cfg.NetworkState,
-		msgRec:       msgRec,
-		msgSend:      msgSend,
+		msgRec:       cfg.MsgRec,
+		msgSend:      cfg.MsgSend,
```

Comment:
an error check in case these are nil would be cool

Reply:
will add :+1: